Integrations/GoogleCloudStorage/GoogleCloudStorage.py

```diff
@@ -0,0 +1,566 @@
+    bucket_name = demisto.args()["bucket_name"]
+    entity = demisto.args()["entity"]
+
+    client = init_storage_client()
+    acl = client.get_bucket(bucket_name).acl
+    if not acl.has_entity(entity):
+        return_error("Entity {} does not exist in the ACL of bucket {}".format(entity, bucket_name))
+
+    delete_acl_entry(acl, entity)
+
+    demisto.results({
+        "Type": entryTypes["note"],
+        "ContentsFormat": formats["text"],
+        "Contents": "Removed entity {} from ACL of bucket {}".format(entity, bucket_name)
+    })
+
+
+''' Object policy (ACL) '''
+
+
+def get_blob_acl(bucket_name, blob_name):
+    client = init_storage_client()
+    bucket = client.get_bucket(bucket_name)
+    blob = storage.Blob(blob_name, bucket)
+    return blob.acl
+
+
+def gcs_list_bucket_object_policy():
+    bucket_name = demisto.args()["bucket_name"]
+    blob_name = demisto.args()["object_name"]
+
+    acl = get_blob_acl(bucket_name, blob_name)
+    acl_entries = get_acl_entries(acl)
+    result = [acl2dict(entry, include_object_name=True) for entry in acl_entries]
+
+    demisto.results({
+        "Type": entryTypes["note"],
+        "ContentsFormat": formats["json"],
+        "Contents": result,
+        "HumanReadable": human_readable_table("ACL policy for object " + blob_name, result),
+        "EntryContext": {ec_key("GCS.BucketObjectPolicy", "Bucket", "Object", "Entity"): result}
+    })
+
+
+def gcs_create_bucket_object_policy():
+    bucket_name = demisto.args()["bucket_name"]
+    blob_name = demisto.args()["object_name"]
+    entity = demisto.args()["entity"]
+    role = demisto.args()["role"]
+
+    acl = get_blob_acl(bucket_name, blob_name)
+    if acl.has_entity(entity):
+        return_error(
+            "Entity {} already exists in the ACL of object {} (use gcs-put-bucket-object-policy to update it)"
+            .format(entity, blob_name))
+
+    set_acl_entry(acl, entity, role)
+
+    demisto.results({
+        "Type": entryTypes["note"],
+        "ContentsFormat": formats["text"],
+        "Contents": "Added entity {} to ACL of object {} with role {}".format(entity, blob_name, role)
+    })
+
+
+def gcs_put_bucket_object_policy():
+    bucket_name = demisto.args()["bucket_name"]
+    blob_name = demisto.args()["object_name"]
+    entity = demisto.args()["entity"]
+    role = demisto.args()["role"]
+
+    acl = get_blob_acl(bucket_name, blob_name)
+    if not acl.has_entity(entity):
+        return_error(
+            "Entity {} does not exist in the ACL of object {} (use gcs-create-bucket-object-policy to create it)"
+            .format(entity, blob_name))
+
+    set_acl_entry(acl, entity, role)
+
+    demisto.results({
+        "Type": entryTypes["note"],
+        "ContentsFormat": formats["text"],
+        "Contents": "Updated ACL entity {} in object {} to role {}".format(entity, blob_name, role)
+    })
+
+
+def gcs_delete_bucket_object_policy():
+    bucket_name = demisto.args()["bucket_name"]
```

Comment:
Are all of those args mandatory? Asking to make sure that we won't get a key error

Reply:
Yep, all mandatory. I made sure to use [] for mandatory args and .get() for optional ones.